mlkem/poly.h

```diff
@@ -215,8 +215,27 @@ ENSURES(ARRAY_IN_BOUNDS(int, k, 0, (MLKEM_N - 1), r->coeffs, 0, 4095));
 
 #define poly_frommsg MLKEM_NAMESPACE(poly_frommsg)
 void poly_frommsg(poly *r, const uint8_t msg[MLKEM_INDCPA_MSGBYTES]);
+
+
 #define poly_tomsg MLKEM_NAMESPACE(poly_tomsg)
-void poly_tomsg(uint8_t msg[MLKEM_INDCPA_MSGBYTES], const poly *r);
+/*************************************************
+ * Name:        poly_tomsg
+ *
+ * Description: Convert polynomial to 32-byte message
+ *
+ * Arguments:   - uint8_t *msg: pointer to output message
+ *              - const poly *r: pointer to input polynomial
+ *                Coefficients must be unsigned canonical
+ **************************************************/
+void poly_tomsg(uint8_t msg[MLKEM_INDCPA_MSGBYTES], const poly *r)
+    // clang-format off
```

Comment:
Hmmm. I would find that odd in case I want to cut/paste whole lines of text (Ctrl-K in EMACS, right?) to move a line of code but NOT the // clang-format off line which stands alone...

Reply:
I think the readability is more important than the editing experience.

Reply:
I am willing to disagree-and-commit on this one. I think a separate PR should do this to all the code once and for all.